python/paddle/fluid/executor.py

```diff
@@ -450,9 +450,13 @@ class Executor(object):
           import numpy
           import os
 
-          use_cuda = True
-          place = fluid.CUDAPlace(0) if use_cuda else fluid.CPUPlace()
-          exe = fluid.Executor(place)
+          # Set place explicitly.
+          # use_cuda = True
+          # place = fluid.CUDAPlace(0) if use_cuda else fluid.CPUPlace()
+          # exe = fluid.Executor(place)
+
+          # place=None, Paddle chooses the device.
```

Comment:
"# place=None, Paddle chooses the device."

"If you don't set place, PaddlePaddle set the default device"

Reply:
Done.

Reply:
PaddlePaddle set**s** ？

Reply:
Done.